resources/css/bem/osu-page.less

```diff
@@ -237,9 +237,10 @@
 
   &--notification-banners {
     @media @desktop {
-      position: relative;
-      height: 0;
+      position: absolute;
```

Comment:
needs to be updated for mobile as well 
![image](https://user-images.githubusercontent.com/1694544/218385891-73a403e6-3e3f-4943-9744-548cb6b8c563.png)